src/app/Leaderboard.tsx

```diff
@@ -10,26 +10,29 @@ type LeaderboardProps = {
 }
 
 export default function Leaderboard(props: LeaderboardProps) {
-    return <div className="leaderboard">
+    return <div className="leaderboard clean-scrollbar">
         <h2 className="leaderboard-title">Leaderboard</h2>
         <table>
             <tbody>
             {props.data.map((entry, index) => {
                 return <tr key={index} className="leaderboard-entry">
-                    <td>{index + 1}</td>
-                    <td>{(() => {
-                        const sliced = entry.country.name.slice(0, 20)
-                        return sliced.length === 20 ? sliced + "..." : sliced
-                    })()}</td>
-                    <td><img
-                        className="leaderboard-entry-flag"
-                        src={`static/countries/svg/${entry.country.code.toLowerCase()}.svg`}
-                        alt={`flag of ${entry.country.name}`}
-                        title={entry.country.name}
-                    />
-                    </td>
-                    <td>{entry.tiles}</td>
-                </tr>
+                            <div>
```

Comment:
there is an issue with the HTML semantics here. `div` cannot be a child of `tr`. 

Consider using `span`'s: 

```jsx
                    <td>
                        <span>{index + 1}</span>
                        <span><img
                            className="leaderboard-entry-flag"
                            src={`static/countries/svg/${entry.country.code.toLowerCase()}.svg`}
                            alt={`flag of ${entry.country.name}`}
                            title={entry.country.name}
                        />
                        </span>
                        <span>{(() => {
                            const sliced = entry.country.name.slice(0, 20)
                            return sliced.length === 20 ? sliced + "..." : sliced
                        })()}</span>
                    </td>

                    <td className="leaderboard-tiles">{entry.tiles}</td>
                </tr>
```